src/core/api/message_api.cpp

```diff
@@ -123,6 +123,13 @@ otError otMessageQueueEnqueue(otMessageQueue *aQueue, otMessage *aMessage)
     return queue->Enqueue(*message);
 }
 
+otError otMessageQueueEnqueueAtFront(otMessageQueue *aQueue, otMessage *aMessage)
```

Comment:
nit: prefer to keep consistent naming - choose "head" or "front" and stick with it throughout :)

Reply:
Good point. Will change it.

Reply:
Updated in the new pushed commit.